internal/store/postgres/metaschemas/defaultGroupSchema.json

```diff
@@ -0,0 +1,18 @@
+{
+    "title":"group metadata",
+    "description":"JSON-schema for validating group metadata",
+    "type":"object",
+    "properties":{
+       "label":{
+          "title":"Label",
+          "description":"Additional context about the group",
+          "type":"string"
```

Comment:
labels should be a kv map(object).

Reply:
Can you give details of what nested properties should be? Or should I allow any key:value pair in labels field?